src/uiSelectMultipleDirective.js

```diff
@@ -258,7 +258,7 @@ uis.directive('uiSelectMultiple', ['uiSelectMinErr','$timeout', function(uiSelec
 
         if(caretPosition > 0 || ($select.search.length && key == KEY.RIGHT)) return false;
 
-        $select.close();
+        $select.close(null, true);
```

Comment:
It's a real shame you can't use named parameters here. You could with some of this

```js
if (skipFocusser.skipFocusser != null || skipFocusser.forceClearSearchInput != null) {
  forceClearSearchInput = skipFocusser.forceClearSearchInput
  skipFocusser = skipFocusser.skipFocusser
}
```

but I think it's likely so brittle and error prone it's not worth it.